ecs-init/docker/docker_test.go

```diff
@@ -237,8 +237,8 @@ func validateCommonCreateContainerOptions(opts godocker.CreateContainerOptions,
 
 	hostCfg := opts.HostConfig
 
-	if len(hostCfg.Binds) != 10 {
-		t.Errorf("Expected exactly 10 elements to be in Binds, but was %d", len(hostCfg.Binds))
+	if len(hostCfg.Binds) != 13 {
```

Comment:
Is there a better way to express this expectation?

Reply:
For a unit test? I think this is reasonable when combined with the assertion later on.

Reply:
@petderek I'm okay with this value here in the unit test, but it is a magic number that you have to know about and reconsider when it causes your tests to fail - it isn't a well defined number with its logic explained. This is the second or third time I've seen this change and, without context, wondered why we should assert that it was 13 (other than watching the test fail, and thinking 'oh yeah!'). I was hoping we had a better way to establish what this ought to be with the volume mount information at hand. It would be great to call out the conditions that it would change if fixing the test isn't practical.

Reply:
Made this magic number a constant and added a note about it.

Reply:
Awesome, thanks for adding that!